wcfsetup/install/files/lib/data/moderation/queue/ModerationQueueAction.class.php

```diff
@@ -110,8 +112,66 @@ public function validateGetOutstandingQueues()
      * Returns a list of outstanding queues.
      *
      * @return  string[]
+     * @deprecated 5.5
```

Comment:
Please add a short explanation blurb: `5.5 - Use foo instead.` or `5.5 - This was tightly bound to the old and removed userpanel implementation.`.